src/NorthwindWebsite/Filters/LoggingActionFilter.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Mvc.Filters;
+using NorthwindWebsite.Core.ApplicationSettings;
+
+namespace NorthwindWebsite.Filters;
+
+public class LoggingActionFilter : IActionFilter
+{
+    private readonly ILogger<LoggingActionFilter> _logger;
+    private readonly AppSettings _appSettings;
+    private readonly bool _loggingEnabled;
+
+    public LoggingActionFilter(
+        ILogger<LoggingActionFilter> logger,
+        AppSettings appSettings)
+    {
+        _logger = logger;
+        _appSettings = appSettings;
+        _loggingEnabled = appSettings.SerilogConfiguration.ActionsLogging;
+    }
+
+    public void OnActionExecuting(ActionExecutingContext context)
+    {
+        if (_loggingEnabled)
+        {
+            _logger.LogWarning(string.Format("Action '{0}' started...", context.ActionDescriptor.DisplayName));
```

Comment:
1. Let's rever this condition to avoid deep nesting in code:
```suggestion
           if (!_loggingEnabled)
           {
               return;
           }
           
            _logger.LogWarning(string.Format("Action '{0}' started...", context.ActionDescriptor.DisplayName));
```
2. I am not sure you need to use `warn` type, it req/res parameters are usually logged as normal info log.

Reply:
Normally, yes, Log Level Infomation is used. 
Log Level Warning has been used since the Logging task to to filter out ASP.NET autologs anmd separate them from the logs I added. For development purposed.

Reply:
Well, logging incoming actions is also used for PROD purposes for further DevOps or SysAdmins Network stats. If you want to write it to different file, you may configure it with `Debug` mode.